src/modules/mc_att_control/mc_att_control.hpp

```diff
@@ -99,7 +99,7 @@ class MulticopterAttitudeControl : public ModuleBase<MulticopterAttitudeControl>
 
 	uORB::SubscriptionInterval _parameter_update_sub{ORB_ID(parameter_update), 1_s};
 
-	uORB::Subscription _hover_thrust_estimate_sub{ORB_ID(hover_thrust_estimate)};
+	uORB::Subscription _hover_thrust_slewrate_estimate_sub{ORB_ID(hover_thrust_estimate)};
```

Comment:
```suggestion
	uORB::Subscription _hover_thrust_estimate_sub{ORB_ID(hover_thrust_estimate)};
```
It's still the hover thrust estimate, not the hover thrust slew-rate estimate, right?

Reply:
That is right